batch-models/src/main/scala/org/ekstep/analytics/dashboard/report/user/UserReportModel.scala

```diff
@@ -94,7 +94,8 @@ object UserReportModel extends IBatchModelTemplate[String, DummyInput, DummyOutp
         col("additionalProperties.externalSystem").alias("External_System"),
         col("additionalProperties.externalSystemId").alias("External_System_Id"),
         col("userOrgID").alias("mdoid"),
-        col("Report_Last_Generated_On")
+        col("Report_Last_Generated_On"),
+        from_unixtime(col("userOrgCreatedDate"), "dd/MM/yyyy").alias("MDO_Created_On")
```

Comment:
Shouldnt the userOrgCreatedDate be before the ReportLastGeneratedOn column @FsherinP

Reply:
No, confirmed with Himanshu and corrected in Story as it will be the last column